.github/workflows/deploy-app.yml

```diff
@@ -39,9 +39,10 @@ jobs:
         key: ${{ secrets.KEY }}
         passphrase: ${{ secrets.PASSPHRASE }}
         script: |
-          become mismatch-finder rm -rf mismatch-finder-repo
-          cp -rv mismatch-finder-repo/ ~tools.mismatch-finder/
-          become mismatch-finder take mismatch-finder-repo
-          become mismatch-finder cp .env mismatch-finder-repo/
-          become mismatch-finder mkdir -p mismatch-finder-repo/storage/app/allowlist
-          become mismatch-finder cp uploaders.txt mismatch-finder-repo/storage/app/allowlist/
\ No newline at end of file
+          # Make sure ~tools.mismatch-finder/mismatch-finder-repo is group writable
+          become mismatch-finder chmod -R g+rwx ~tools.mismatch-finder/mismatch-finder-repo
+          # Change group of ~/mismatch-finder-repo (including symlinks) to tools.mismatch-finder
+          chgrp --no-dereference -R tools.mismatch-finder ~/mismatch-finder-repo
+          rsync -rlgD --delete --delay-updates --exclude '.nfs*' --exclude mismatch-finder-repo/.env --exclude mismatch-finder-repo/storage/app/ ~/mismatch-finder-repo ~tools.mismatch-finder/
+          # take aborts recursion whenever it encounters a symlink, thus we use find+xargs to make sure all folders and file ares handled.
+          find ~tools.mismatch-finder/mismatch-finder-repo -type d,f \! -user tools.mismatch-finder -print0 | become mismatch-finder xargs -r --null take 2>&1 | { grep -vF 'will not follow or touch symlinks' || true; }
```

Comment:
This line confuses me a bit. I'm not sure why we would need to follow symlinks in this case? Did it cause nay issues before?

Reply:
We don't need to follow symlinks… but the problem is that `take` stops recursing through the directories in case it encounters a symlink (it doesn't just skip it). So we can't just run take on all of `mismatch-finder-repo` at once, as it wont process all files.

Reply:
Gotcha, thanks!